templates/README.template

```diff
@@ -22,91 +23,200 @@ npm install --save-dev ${package}
 ```
 
-<h2 align="center">Maintainers</h2>
+### `Option 3`
 
-```bash
-https://api.github.com/users/MAINTAINER
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+
+<!-- isLoader ? use(this) : delete(isPlugin) -->
+**webpack.config.js**
+```js
+{
+  test: /\.ext$/,
+  use: [
+    {
+      loader: `${name.toLowerCase()}-loader`,
+      options: { option3: '' }
+    }
+  ]
+}
+```
+
+<!-- isPlugin ? use(this) : delete(isLoader) -->
+**webpack.config.js**
+```js
+new `${name}`Plugin({ option3: '' })
+```
+
+<h2 align="center">Examples</h2>
+
+### `Example 1`
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+**webpack.config.js**
+```js
+// Example setup here..
 ```
 
+**file.ext**
+```js
+// Source code here...
+```
+
+**bundle.js**
+```js
+// Bundle code here...
+```
+
+<h2 align="center">Maintainers</h2>
+
+<!-- 4 Maintainers per row -->
 <table>
   <tbody>
     <tr>
       <td align="center">
-        <a href="https://github.com/">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/5419992?v=3&s=150">
-          </br>
-          Name
-        </a>
-      </td>
-      <td align="center">
-        <a href="https://github.com/">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/5419992?v=3&s=150">
+        <a href="https://github.com/bebraw">
+          <img width="150" height="150" src="https://github.com/bebraw.png?v=3&s=150">
           </br>
-          Name
+          Juho Vepsäläinen
         </a>
       </td>
       <td align="center">
-        <a href="https://github.com/">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/5419992?v=3&s=150">
+        <a href="https://github.com/d3viant0ne">
+          <img width="150" height="150" src="https://github.com/d3viant0ne.png?v=3&s=150">
           </br>
-          Name
+          Joshua Wiens
         </a>
       </td>
       <td align="center">
-        <a href="https://github.com/">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/5419992?v=3&s=150">
+        <a href="https://github.com/sapegin">
+          <img width="150" height="150" src="https://github.com/sapegin.png?v=3&s=150">
           </br>
-          Name
+          Artem Sapegin
         </a>
       </td>
       <td align="center">
-        <a href="https://github.com/">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/5419992?v=3&s=150">
+        <a href="https://github.com/michael-ciniawsky">
+          <img width="150" height="150" src="https://github.com/michael-ciniawsky.png?v=3&s=150">
           </br>
-          Name
+          Michael Ciniawsky
         </a>
```

Comment:
> 1.) Tobias specifically asked that there be no more than 5 listed, ever.

With Alexander added we are full then :): Will do...

> 2.) Asked not to be listed under the maintainers section.

You mean Tobias here ?